tasks/fr/insee.py

```diff
@@ -277,13 +277,28 @@ def columns(self):
             type='Text',
             name='DCOMIRIS',
             description='Full Code IRIS. Result of the concatenation of DEPCOM and IRIS attributes. ',
-            weight=0,
+            weight=1,
```

Comment:
LaCaixa changes are relevant. GEOM_REF for INSEE is actually weight=0 in the france-boundary-names branch... not sure why it's 1 here and how it even got into this branch. The france-boundary-names branch will need to be merged first then.